src/controllers/helpers/pagination.rs

```diff
@@ -580,15 +610,16 @@ mod tests {
         use chrono::naive::serde::ts_microseconds;
         seek! {
             pub(super) enum Seek {
-                Id(i32)
-                New(#[serde(with="ts_microseconds")] chrono::NaiveDateTime, i32)
-                RecentDownloads(Option<i64>, i32)
+                Id{id: i32}
+                New{#[serde(with="ts_microseconds")] dt: chrono::NaiveDateTime, id: i32}
+                RecentDownloads{ downloads: Option<i64>, id: i32 }
```

Comment:
This could use some manual formatting, since `rustfmt` obviously isn't getting in here.

Reply:
I've managed to figure out how to `rustfmt` can be used for this:

1. we need to have valid Rust code inside the macro invocation, but the implementation was missing trailing commas for the enum variants
2. it needs to be invoked as `seek!(...);` instead of `seek! { ... }` and apparently that enables rustfmt to format the invocation content (see https://users.rust-lang.org/t/rustfmt-skips-macro-arguments/74807/4)

I've added two commits to the PR for the above.

Reply:
Thank you for your help. I appreciate it.